### PR TITLE
cleaning up chapter 5

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: Chapter_5/emr/README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
-This folder contains below setup instruction files:
-- [EC2 single-node instance setup](ec2.md) 
-- [EMR setup](EMR.md)
\ No newline at end of file
+### Setting up an EMR Cluster for ETL
+
+// ToDo: Rearrange sequence & add an introduction
```
Comment (ljj7975) [inline Chapter_5/emr/README.md]: @leninworld

File: Chapter_5/glue/README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
-### Glue
+## Creating a Glue Job for ETL
 
+TODO: introduction paragraph
```
Comment (ljj7975) [inline Chapter_5/glue/README.md]: @leninworld

File: Chapter_5/sagemaker/sagemaker_studio.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-### Introduction to Sagemaker Studio 
+### Introduction to Sagemaker Studio
+
+TODO: add the actual introduction to SageMaker Studio
```
Comment (ljj7975) [inline Chapter_5/sagemaker/sagemaker_studio.md]: @leninworld

File: Chapter_5/sagemaker/README.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
+## Utilizing SageMaker for ETL
 
-- Introduction to **Sagemaker Studio** with a document [here](./sagemaker_intro.md).
-- [Example Jupyter Notebook](./cifar10-tensorflow.ipynb) using [ResNet](https://arxiv.org/abs/1512.03385) with [CIFAR 10](https://www.cs.toronto.edu/~kriz/cifar.html).
-  This code has been revised after being taken from the AWS sample [here](https://aws.amazon.com/getting-started/hands-on/train-tune-deep-learning-model-amazon-sagemaker/) in order to run successfully.
+[SageMaker](https://aws.amazon.com/sagemaker/) is an end-to-end service for machine learning. You can configure SageMaker to handle data processing, model development with notebooks, model training, as well as deployment of models to a production setting.
+
+In this repository,
+* we explain [how to set up a SageMaker Studio](./sagemaker_studio.md), a web-based development environment for SageMaker.
+* [A sample model training implementation on SageMaker](./cifar10-tensorflow.ipynb). The model is [ResNet](https://arxiv.org/abs/1512.03385) trained with [CIFAR 10](https://www.cs.toronto.edu/~kriz/cifar.html) for image classification.
```
Comment (ljj7975) [inline Chapter_5/sagemaker/README.md:7]: @leninworld  the notebook looks unpolished (has error message as well)

Can you make sure the code is cleaned up and runnable?

File: Chapter_5/emr/README.md (hunk 1 above)
Comment (leninworld) [reply Chapter_5/emr/README.md]: done

File: Chapter_5/glue/README.md (hunk 2 above)
Comment (leninworld) [reply Chapter_5/glue/README.md]: done

File: Chapter_5/sagemaker/sagemaker_studio.md (hunk 3 above)
Comment (leninworld) [reply Chapter_5/sagemaker/sagemaker_studio.md]: done.

File: Chapter_5/sagemaker/README.md (hunk 4 above)
Comment (leninworld) [reply Chapter_5/sagemaker/README.md:7]: The last few blocks are supposed to be removed (and forgotten) and are now removed. The rest of the code was already tested 3 times for cifar and should be fine. good.

File: Chapter_5/emr/README.md [hunk 5]
```diff
@@ -1,55 +1,75 @@
-### Setting up an EMR Cluster for ETL
+### EMR Setup
```
Comment (ljj7975) [inline Chapter_5/emr/README.md]: Can you keep the heading as is.

I am trying to align them with the headings in the book.

`### Setting up an EMR Cluster for ETL`

File: Chapter_5/emr/README.md [hunk 6]
```diff
@@ -1,55 +1,75 @@
...
+
```
Comment (ljj7975) [inline Chapter_5/emr/README.md]: A short introduction describing "Setting up an EMR Cluster for ETL"

File: Chapter_5/emr/README.md [hunk 7]
```diff
@@ -1,55 +1,75 @@
-### Setting up an EMR Cluster for ETL
+### EMR Setup
+ 
+### Prerequisite:
```
Comment (ljj7975) [inline Chapter_5/emr/README.md]: these headings should have 4 #s to be distinguished from the main heading

File: Chapter_5/glue/README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ## Creating a Glue Job for ETL
 
-TODO: introduction paragraph
+We will show with step-by-step on how to create a glue job for ETL.
```
Comment (ljj7975) [inline Chapter_5/glue/README.md]: In this file, we describe how to create a Glue job for ETL step by step.

Comment (leninworld) [reply Chapter_5/glue/README.md]: done

File: Chapter_5/emr/README.md (hunk 7 above)
Comment (leninworld) [reply Chapter_5/emr/README.md]: done

File: Chapter_5/emr/README.md (hunk 6 above)
Comment (leninworld) [reply Chapter_5/emr/README.md]: done

File: Chapter_5/emr/README.md (hunk 5 above)
Comment (leninworld) [reply Chapter_5/emr/README.md]: done

**Pull request merged**